<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Pages">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-30</y>
      <width>931</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="main">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="ShowScores">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>450</y>
        <width>801</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>ScoreBoard</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Play">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>801</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PlayerName">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>131</y>
        <width>741</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="maxLength">
       <number>10</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Enter Name:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="GameWidth">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>290</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>40</number>
      </property>
      <property name="value">
       <number>40</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="GameHeigth">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>290</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>290</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Heigth</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Width</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>290</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Seed value</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="SemenValue">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>290</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-10000</number>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Scores">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="BackToMain">
      <property name="geometry">
       <rect>
        <x>54</x>
        <y>520</y>
        <width>731</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Main</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>560</y>
        <width>721</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color:red;
</string>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="scoreBoard">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>731</width>
        <height>481</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="itemAlignment">
       <set>Qt::AlignHCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gameTab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>881</width>
        <height>641</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
